docs/content/interface/toolbar.md

```diff
@@ -0,0 +1,96 @@
+# Toolbar
+
+![whole toolbar](../images/interface/toolbar/whole.png)
+
+The Toolbar is found above the editor and is split into five sections:
+
+1. Emphasis
+2. Header & Blocks
+3. Links & Images
+4. Larger Blocks & Emojis
+5. Settings
+
+## Emphasis
+
+![Emphasis toolbar](../images/interface/toolbar/emphasis.png)
+
+These buttons are able to either surround selected text with the necessary symbols to emphasize it or insert the symbols
+without any content between them to be filled out by you. The icons on the buttons should be rather easy to identify.
+Nonetheless, here is their functionality:
+
+1. bold e.g `**text**`
+2. italic e.g `*text*`
+3. underscore e.g `_text_`
+4. strikethrough e.g `~~text~~`
+5. subscript e.g `~text~`
+6. superscript e.g `^text^`
+
+## Header & Blocks
+
+![Header & Blocks toolbar](../images/interface/toolbar/blocks.png)
+
+These buttons all work a bit different so, we will describe each on separately:
+
+1. **Heading**  
+   Adds `# ` in front of the line. If the line is already starting with a `#` adds just `#`.
+2. **Code Blocks**  
+   Adds ` ``` ` around the selected text or if no text is selected just adds two lines with ` ``` ` and an empty line
+   between them.
+3. **Quote**  
+   Adds `> ` in front of the line.
+4. **Unordered List**  
+   Ads `- ` in front of the line.
+5. **Ordered List**  
+   Adds `1. ` in front of the line.
+6. **ToDo List**  
+   Adds `- [ ] ` in front of the line.
+
+## Links & Images
+
+![Links & Images toolbar](../images/interface/toolbar/links.png)
+
+These buttons add the corresponding syntax either in the text or if some text is selected it will either put that as the
+link, if it thinks that the selected text is a link, or as the alt text.
+
+1. **Link**  
+   Adds the link template `[](https://)` (For more information
+   see [the markdown link documentation](https://daringfireball.net/projects/markdown/syntax#link))
```

Comment:
As we use CommonMark, what about referring to [their spec](https://spec.commonmark.org/current/)?

Reply:
It seems I can't deep link to the link section if I also want to link to the current (and no specific version). What do you think would be the best approach here?

Reply:
We should link to a specific version of the spec, this will prevent link breakage and will also make it clear if we need to update the docs.

Reply:
done